discovery-medicine.csl

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Discovery Medicine</title>
+    <title-short>DM</title-short>
+    <id>http://www.zotero.org/styles/discovery-medicine</id>
+    <link href="http://www.zotero.org/styles/discovery-medicine" rel="self"/>
+    <link href="http://www.zotero.org/styles/brain" rel="template"/>
+    <link href="http://www.discoverymedicine.com/about/information-for-authors-of-discovery-medicine/" rel="documentation"/>
+    <author>
+      <name>O'Brien Patrick, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="medicine"/>
+    <issn>1539-6509</issn>
+    <eissn>1944-7930</eissn>
+    <updated>2017-01-06T10:47:39+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" delimiter=", " initialize-with=". " delimiter-precedes-last="never"/>
+      <et-al font-style="italic"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <choose>
+          <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+            <text variable="title" form="short" font-style="italic"/>
+          </if>
+          <else>
+            <text variable="title" form="short" quotes="true"/>
+          </else>
+        </choose>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="issued-year">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="citation-locator">
+    <group>
+      <label variable="locator" form="short"/>
+      <text variable="locator" prefix=" "/>
+    </group>
+  </macro>
+  <macro name="author">
+    <names variable="author" suffix=". ">
+      <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always"/>
+    </names>
```

Comment:
I always make sure the `substitute` elements in author date styles conform between author and author-short macro. Otherwise, how would authors find the bib entry corresponding to a citation. In other words, add editor, translator, and title as substitute elements here. 
With the editor and translator now options, I would then also suggest adding `<label form="short" prefix=" (" suffix=")"/>`